lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1755,6 +1755,7 @@ sub test_uploadlog_list {
     # get a list of uploaded logs
     my ($self) = @_;
     my $testresdir = $self->result_dir();
+    return [] unless $testresdir;
```

Comment:
Isn't `return [] unless my $testresdir = $self->result_dir();` more your style?

Reply:
yeah, I like that :) Done.